cmd/aries-agent-mobile/pkg/wrappers/rest/rest.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package rest
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+)
+
+type httpClient interface {
+	Do(r *http.Request) (*http.Response, error)
+}
+
+func makeHTTPRequest(httpClient httpClient, method, url, token string) (string, error) {
+	req, err := http.NewRequest(method, url, nil)
+	if err != nil {
+		return "", fmt.Errorf("failed to create a new http request for [%s]: %v", url, err)
+	}
+
+	req.Header.Set("Content-Type", "application/json")
+
+	if token != "" {
+		req.Header.Set("Authorization", "Bearer "+token)
+	}
+
+	response, err := httpClient.Do(req)
+	if err != nil {
+		return "", fmt.Errorf("error while making request to [%s]: %v", url, err)
+	}
+
+	defer func() {
+		e := response.Body.Close()
+		if e != nil {
+			// TODO use `/pkg/common/log` package instead
+			fmt.Print(fmt.Errorf("failed to close response body: %v", e))
```

Comment:
Please fix this TODO, we should use proper log package. See other packages using log examples.